src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -2306,6 +2398,33 @@ bool Scene3D::OnViewAngle(const msgs::Vector3d &_msg,
   return true;
 }
 
+/////////////////////////////////////////////////
+bool Scene3D::OnMoveToPose(const msgs::Pose &_msg, msgs::Boolean &_res)
```

Comment:
I think we should be making use of the [GUICamera](https://github.com/ignitionrobotics/ign-msgs/blob/master/proto/ignition/msgs/gui_camera.proto) message for these things. But maybe this will need some deeper refactoring, so probably this is too much to do now...

Another thing I think would be good to include in the message is the animation time. I imagine some use cases may need the camera to teleport directly to the target pose without an animation.

Reply:
Changed to GUICamera in 251b573c3e83ee444d4d136a08eeb5e514eae91f.
Do you want to capture the animation time idea in an issue?

Reply:
Sure, I can ticket an issue.

Do you think now it's weird that the service is called `/gui/camera/pose` but the message can take more than just pose though?

Reply:
I think it's fine. I think the idea would be to add new parameters to the GUICamera message, which we couldn't do with the Pose message.